## Mothur

Mothur is a bioinformatics platform specially designed for microbiome research. Developed by the Schloss lab at the University of Michigan, it offers a suite of software tools for sequence data, including amplicon sequence variants (ASVs), operational taxonomic units (OTUs), metagenomics, metatranscriptomics, and Fungi ITS sequences. With a forum-supported community, Mothur makes it easier to analyze high-throughput community sequencing data.

### Core Features 

1. **Flexible Data Handling**: Mothur can process a broad range of raw data, including Illumina, 454, SOLiD, IonTorrent, PacBio, and Sanger.

2. **Data Analysis**: Mothur provides a variety of methods for community type analyses such as alpha and beta diversity, rarefaction, and species richness estimation.

3. **Batch Processing and Automation**: Mothur supports batch files, allowing for automation of common workflows which increase reproducibility and decrease time spent on data processing.

4. **Command-line Interface**: It is driven by a command-line interface, enabling users to write scripts and automate their analyses.

5. **Tutorial and Community Support**: Mothur has a proactive community that helps in understanding and troubleshooting the platform.

### TAGS

Software: Bioinformatics, Genome Analysis, Microbiome, High-Throughput Sequencing

Research Discipline: Bioinformatics, Microbiology, Genomics

Research Area: Microbiome Analysis, Metagenomics, Community Sequencing

Software Class: Bioinformatics Tools

Software Type: Open Source 

[Go to the Mothur Official Website](https://www.mothur.org)

[View the Mothur Documentation](https://mothur.org/wiki/commands/)

[Mothur Tutorials](https://mothur.org/wiki/miseq_sop/)

--------------------------------------
