# SAMBLASTER

SAMBLASTER is a fast and flexible program that allows for the removal of duplicates and splitting of read alignments from SAM and BAM files. Created specifically for next-generation sequencing data, SAMBLASTER's primary goal is to mark duplicates and extract discordant and split alignment reads for use in detecting genetic variants such as Single Nucleotide Variants (SNVs), indels, and structural variants (SVs).

## Core Features

- **Duplicate Marking**: SAMBLASTER can mark duplicates in a SAM file coming from a high-throughput sequencing experiment.

- **Efficient Splitting and Extraction**: The software can extract the reads causing split-read alignments with significant computational efficiency.

- **Compatibility**: It can also output discordant read pairs/clusters for use in detecting rearrangements.

- **Low Memory Footprint**: SAMBLASTER runs entirely from the command line with a low-memory footprint, improving its feasibility for larger scale research projects.

## Tags

Software, Bioinformatics, Genomics, Open-Source, Duplicate Marking, Sequence Alignment, Genomic Variants

### Research Disciplines

Genomics, Bioinformatics, Medical Science, Biological Science

### Research Area

Genetic Variants, DNA Sequencing

### Software Class

Bioinformatics Software

### Software Type

Open Source

## Links

You can find the SAMBLASTER software on its GitHub page at the following link:

[SAMBLASTER on GitHub](https://github.com/GregoryFaust/samblaster)

## Additional Web URLs

- SAMBLASTER Documentation: [Readme on GitHub](https://github.com/GregoryFaust/samblaster#readme)
- Tutorial materials specific to SAMBLASTER are not readily available, but several genetics and bioinformatics coursework and tutorials provide relevant information and training.

--------------------------------------
